test/linter/test_buf_lint.vader

```diff
@@ -0,0 +1,22 @@
+Before:
+  call ale#assert#SetUpLinterTest('proto', 'buf-lint')
+  call ale#test#SetFilename('test.proto')
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(The default command should be correct):
+  AssertLinter 'buf-lint',
```

Comment:
This first argument is the binary name so this should be "buf" instead of "buf-lint".